scripts/dev-agent.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+. setup-rancher-path.sh
```

Comment:
Can you move this to line 7.  If you run `./scripts/dev-agent.sh` it will fail because the script is not yet in `./scripts` folder.

Reply:
I wanted to get this merged real quick so I just patched this in #180